parquet-cli/pom.xml

```diff
@@ -110,6 +110,12 @@
       <artifactId>avro</artifactId>
       <version>${avro.version}</version>
     </dependency>
+    <dependency>
+      <groupId>org.tukaani</groupId>
```

Comment:
Does it have any license issue? And what about its dependency?

Reply:
I think it has no license issue, since [its Java implementation](https://tukaani.org/xz/java.html) is [public domain](https://github.com/tukaani-project/xz-java/blob/v1.9/COPYING), which is classified into [Category A](https://www.apache.org/legal/resolved.html#handling-public-domain-licensed-works). It also doesn't seem to have any dependent libraries.

As other examples, Druid, HBase and Spark already include it.
https://github.com/apache/druid/blob/druid-27.0.0/pom.xml#L527-L531
https://github.com/apache/hbase/blob/rel/2.5.5/pom.xml#L1470-L1474
https://github.com/apache/spark/blob/v3.4.1/pom.xml#L1489-L1493